codegen/smithy-aws-typescript-codegen/src/main/java/software/amazon/smithy/aws/typescript/codegen/AddBuiltinPlugins.java

```diff
@@ -196,13 +201,19 @@
                 RuntimeClientPlugin.builder()
                         .withConventions(AwsDependency.MIDDLEWARE_HOST_HEADER.dependency, "HostHeader")
                         .build(),
-                RuntimeClientPlugin.builder()
-                        .withConventions(AwsDependency.MIDDLEWARE_SDK_COGNITO_IDENTITY.dependency, "RemoveAuth",
-                                        HAS_MIDDLEWARE)
-                        .operationPredicate((m, s, o) -> testServiceId(s, "Cognito Identity") && SetUtils.of(
-                                    "GetId", "GetOpenIdToken", "GetCredentialsForIdentity"
-                                    ).contains(o.getId().getName())
-                        ).build()
+                //Cognito Identity service doesn't need auth for GetId, GetOpenIdToken, GetCredentialsForIdentity
+                RuntimeClientPlugin.builder()
+                        .withConventions(AwsDependency.MIDDLEWARE_SIGNING.dependency, "AwsAuth", HAS_MIDDLEWARE)
+                        .operationPredicate((m, s, o) -> {
```

Comment:
Can you split this out to its own function? This will be clearer for updating this list in the future if necessary.

Usage:
```java
.operationPredicate(AddBuiltinPlugins::operationUsesAwsAuth)
```

Definition:
```java
private static boolean operationUsesAwsAuth(Model model, ServiceShape service, OperationShape operation) {
    if (testServiceId(s, "Cognito Identity")
        && SetUtils.of("GetId", "GetOpenIdToken", "GetCredentialsForIdentity")
                .contains(o.getId().getName())
    ) {
        return false;
    }
    return true;
}
```